website/docs: capitalization and clarifications (#4948)
goauthentik/authentik: website/docs/installation/docker-compose.md

```shell
AUTHENTIK_PORT_HTTP=80
AUTHENTIK_PORT_HTTPS=443
```

Be sure to run `docker-compose up -d` to rebuild with the new port numbers.

## Startup

Afterwards, run these commands to finish:

```shell
docker-compose pull
docker-compose up -d
```

The `docker-compose.yml` file statically references the latest version available at the time of downloading the compose file, which can be overridden with the `AUTHENTIK_TAG` environment variable.

authentik is then reachable (by default) on port 9000 (HTTP) and port 9443 (HTTPS).

To start the initial setup, navigate to `https://<your server's IP or hostname>:9000/if/flow/initial-setup/`.

There you will be prompted to set a password for the akadmin user (the default user).

## Explanation

:::warning
The server assumes to have local timezone as UTC.
All internals are handled in UTC; whenever a time is displayed to the user in UI it gets localized.
Do not update or mount `/etc/timezone` or `/etc/localtime` in the authentik containers.
This will not give any advantages.
On the contrary, it will cause problems with OAuth and SAML authentication,
e.g. [see this GitHub issue](https://github.com/goauthentik/authentik/issues/3005).
:::

> The Docker-Compose project contains the following containers:

Author: ```suggestion
The Docker Compose project contains the following containers:
```

-   server

    This is the backend service, which does all the logic, plus runs the API and the SSO functionality. It also runs the frontend, hosts the JS/CSS files, and serves the files you've uploaded for icons/etc.

-   worker

    This container executes background tasks, everything you can see on the _System Tasks_ page in the frontend.

-   redis (for cache)

> -   postgresql (default database)

Author: ```suggestion
-   postgresql (default database)

If you've enabled GeoIP, refer to the [GeoIP documentation](../core/geoip#external-updates)
```

AUTHENTIK_EMAIL__FROM=authentik@localhost
```

## Configure for port 80/443

By default, authentik listens on port 9000 for HTTP and 9443 for HTTPS. To change the default and instead use ports 80 and 443, you can set the following variables in `.env`:

```shell
AUTHENTIK_PORT_HTTP=80
AUTHENTIK_PORT_HTTPS=443
```

Be sure to run `docker-compose up -d` to rebuild with the new port numbers.

## Startup

Afterwards, run these commands to finish:

```shell
docker-compose pull
docker-compose up -d
```

The `docker-compose.yml` file statically references the latest version available at the time of downloading the compose file, which can be overridden with the `AUTHENTIK_TAG` environment variable.

authentik is then reachable (by default) on port 9000 (HTTP) and port 9443 (HTTPS).

> To start the initial setup, navigate to `https://<your server>/if/flow/initial-setup/`. If you are using your local host, this will be `http://localhost:9000/if/flow/initial-setup/`.

Reviewer: ```suggestion
To start the initial setup, navigate to `https://<your server's IP or hostname>:9000/if/flow/initial-setup/`.
```

I think very few people will use these instructions to run authentik on the machine they are accessing it from, and those people are most likely doing so for a reason hence I think it makes sense to assume they know how to access it

Author: agree, good point.

There you will be prompted to set a password for the akadmin user (the default user).

## Explanation

:::warning
The server assumes to have local timezone as UTC.
All internals are handled in UTC; whenever a time is displayed to the user in UI it gets localized.
Do not update or mount `/etc/timezone` or `/etc/localtime` in the authentik containers.
This will not give any advantages.
On the contrary, it will cause problems with OAuth and SAML authentication,
e.g. [see this GitHub issue](https://github.com/goauthentik/authentik/issues/3005).
:::

The Docker-Compose project contains the following containers:

-   server

    This is the backend service, which does all the logic, plus runs the API and the SSO functionality. It also runs the frontend, hosts the JS/CSS files, and serves the files you've uploaded for icons/etc.

-   worker

    This container executes background tasks, everything you can see on the _System Tasks_ page in the frontend.

-   redis (for cache)

-   postgresql (default database)

> Additionally, if you've enabled GeoIP, there is a container running that regularly updates the GeoIP database.

Reviewer: ```suggestion
```

This can actually be removed since GeoIP is built-in by default without an extra container, or we can maybe link https://goauthentik.io/docs/core/geoip#external-updates

Author: great, I'll add a link in separate PR.
